types/trusted-types/index.d.ts

```diff
@@ -4,30 +4,32 @@
 //                 Damien Engels <https://github.com/engelsdamien>
 //                 Emanuel Tesar <https://github.com/siegrift>
 //                 Bjarki <https://github.com/bjarkler>
+//                 Sebastian Silbermann <https://github.com/eps1lon>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.1
 
 import * as lib from './lib';
 
 // Re-export the type definitions globally.
 declare global {
-    const TrustedHTML: typeof lib.TrustedHTML;
```

Comment:
This change would make `TrustedHTML` only a type and not a value. Is that what you intended to do? If not, then you can change `const TrustedHTML` to `var TrustedHTML` .